cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/Python.kt

```diff
@@ -1287,6 +1287,8 @@ interface Python {
         val context_expr: ASTBASEexpr by lazy { "context_expr" of pyObject }
         val optional_vars: ASTBASEexpr? by lazy { "optional_vars" of pyObject }
     }
+
+    class Ellipsis(python: PyObject) : AST(python)
```

Comment:
Where should I put it then? We only have a `Python` interface with contains classes that are prefixed with their package name. For example we have `Python.ASTModule` which represents the `ast.Module` class. So following this convention `Ellipsis` should be `Python.Ellipsis`.

Alternately we can put these things in nested interfaces as I do I Go, where I have something like `Go.Ast.FunctionDef`.

Reply:
The same problem also would apply to other classes we need like `complex`. And we should probably have a common base class for a python object that is not `AST`, so this might need more refactoring if we want to do it "correctly".

Reply:
I like the Go approach a lot. This should probably be a separate PR then, correct? I can do it tomorrow, if you don't beat me to it ;)

Reply:
Go for it

Reply:
https://github.com/Fraunhofer-AISEC/cpg/pull/1675